dask_expr/_collection.py

```diff
@@ -1529,9 +1531,14 @@ def to_backend(self, backend: str | None = None, **kwargs):
         -------
         DataFrame, Series or Index
         """
-        from dask.dataframe.io import to_backend
+        from dask_expr._backends import dataframe_creation_dispatch
 
-        return to_backend(self.to_dask_dataframe(), backend=backend, **kwargs)
```

Comment:
The previous code here was just converting to `dask.dataframe` - Definitely not what we want.